src/include/postgres.h

```diff
@@ -805,4 +805,10 @@ extern Datum Float8GetDatum(float8 X);
 #define Float8GetDatumFast(X) PointerGetDatum(&(X))
 #endif
 
+extern PGDLLIMPORT char *log_connection_messages_string;
```

Comment:
Could you please explain, why do we need to define `extern PGDLLIMPORT` for `log_connection_messages_string`? Is it used anywhere else apart from guc.c?

Reply:
true, not used anywhere excep guc.c I've dropped `extern PGDLLIMPORT`